joint_trajectory_controller/doc/parameters_context.yaml

```
@@ -16,3 +14,6 @@ gains: |
   i.e., the shortest rotation to the target position is the desired motion.
   Otherwise  :math:`e = s_d - s` is used, with the desired position :math:`s_d` and the measured
   position :math:`s` from the state interface.
+
+  If you want to turn of the PIDs (open loop control), set the gains to zero.
```

```suggestion
  If you want to turn off the PIDs (open loop control), set the feedback gains to zero.
```
`ff_velocity_scale` has to be set to 1.0 in this case, we probably should change the default value.